easysocial/easysocial/discussion.php

```diff
@@ -78,7 +78,9 @@ public function getGroupDiscussion()
 	{
 		// Init variable
 		$mainframe		=	JFactory::getApplication();
-		$group_id		=	$mainframe->input->get('id', 0, 'INT');
+		$cluster_id		=	$mainframe->input->get('id', 0, 'INT');
+		$type			=	$mainframe->input->get('type', 'group', 'string');
+		$page_id		=	$mainframe->input->get('page_id', 0, 'INT');
```

Comment:
@Pratikshakhandagale remove $page_id from here. We are already get the id as cluster id here.